pkg/cmd/root.go

```diff
@@ -137,6 +137,10 @@ func ensureConfigDefaults() {
 		tasklet.MustRun(tasklet.Runner{
 			StartMsg: "Updating configfile to include defaults",
 			Runner: func(_ tasklet.TaskletContext) error {
+				// ensure .config/nitric exists
+				err := os.MkdirAll(utils.NitricConfigDir(), os.ModePerm)
+				cobra.CheckErr(err)
```

Comment:
```suggestion
				if err!=nil{
				return err
				}
```
just so the error will be printed nicely